src/utils/boardImportUtils.ts

```diff
@@ -103,4 +104,25 @@ const uploadAction = (
   return promise;
 };
 
-export { openFromFileSystem, importAssetsToS3 };
+const fileMimeExtensions: { [key: string]: string[] } = {
+  "image/jpeg": [".jpg", ".jpeg"],
+  "image/png": [".png"],
+  "image/svg+xml": [".svg"],
+  "image/gif": [".gif"],
+};
+
+const allowedMimeTypes =
+  envs?.TLDRAW__ASSETS_ALLOWED_MIME_TYPES_LIST?.split(",") || [];
+
+const openAssetsFromFileSystem = async () => {
+  const extensions = allowedMimeTypes.flatMap(
+    (mimeType) => fileMimeExtensions[mimeType] || [],
```

Comment:
Well, if new mime type is added it would need to be added to this mapper too.
Extensions as env var are the easiest, but against devops suggestions. We did that before and were told to change it.
We could maybe use a mapper from external library but that's somewhat tricky too.
This lib for example: https://www.npmjs.com/package/mime
gives .jpe extensions for image/jpeg type which will not work as tldraw does additional extension check inside its own code...

Reply:
There are some licensing issues as far as I'm aware, not sure if we will be upgrading to v2.
And if we ever do, this entire repo will need to be rewritten anyway as there are huge breaking changes and almost nothing of current code will be useful.

Reply:
And in current code we have to use our own mime check because tldraw accepts video formats by default, but because its video player is buggy we were told to disable video upload.